Lib/py_compile.py

```diff
@@ -137,6 +137,10 @@ def compile(file, cfile=None, dfile=None, doraise=False, optimize=-1):
     except FileExistsError:
         pass
     source_stats = loader.path_stats(file)
+    sde = os.environ.get('SOURCE_DATE_EPOCH')
```

Comment:
Could the variable name be a bit more descriptive? We avoid abbreviations like this in the stdlib when it isn't obvious what it stands for.